pages/clusters/overview.md

```diff
@@ -12,7 +15,21 @@ Clusters encapsulate groups of agents and pipelines, enabling the following:
 * Individual users or teams can maintain their own clusters. Cluster maintainers can manage queues and agent tokens and add and remove pipelines.
 * Pipelines can be assigned to a cluster, ensuring their builds run only on the agents connected to this cluster. These pipelines can also trigger builds only on other pipelines in the same cluster.
 
-## Enable clusters
+## Queue metrics
+
+Clusters provides additional, easy to access queue metrics that are available only for queues within a cluster.
+See [Cluster queue metrics](../../docs/pipelines/cluster-queue-metrics) for more information.
```

Comment:
When creating relative links, there's no need to traverse up directory structures to reach the page file. In fact, these are not file references (which have underscores separating their words); they're virtual 'relative' page references instead, and are all fixed from `/doc`.

```suggestion
See [Cluster queue metrics](/docs/pipelines/cluster-queue-metrics) for more information.
```

For more information about linking, I've updated the style guide here: https://github.com/buildkite/docs/blob/main/styleguides/markdown-syntax-style.md#internal-links-to-other-pages